### PR TITLE
Feature#5 signup add form validations

### LINKED ISSUE #5: SignUp: Add Form Validations
# REQUIREMENTS
- All fields are full 
- The password must have 8 or more characters with a mix letteres, numbers and symbols. 
- The terms must be accepted.
- The buttons will be enables when all fields are completes.

# DESIGN

### PR DESCRIPTION
**Pull request by tmario1508**
# Description
Form signup with validations
 - First name is required, maximum and minimum characters.
 - Email is required and valid. 
 - Password is required, one capital letter, one symbol, minimum 8 characters. 
 - Accepted terms and conditions.
 - Button SignUp disabled if the fields are not valid.
  
  ## Screenshots 
signup
signup-check
signup-validate

### CODE REVIEW

**Review by CrMa1 — DISMISSED** [review]
Excelent

**Review by Kejoh — DISMISSED** [review]
Good text input solution

File: ReservacionVuelos/src/components/SignUpButtons.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <Text style={signUpStyles.textLogin}>
+        Already have an account? Log In
+      </Text>
+    </View>
+  );
+};
+
+export default SignUpButtons;
+
+const styles = StyleSheet.create({});
```
Comment (YilKanda) [inline ReservacionVuelos/src/components/SignUpButtons.js]: This part of the styles remember to put it in another file or if you don't use it, delete it.

File: ReservacionVuelos/src/components/TextTerms.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <Text>
+      I agree to the
+      <Text style={signUpStyles.textTerms}>Terms and Privacy Policy</Text>
+    </Text>
+  );
+};
+
+export default TextTerms;
+
+const styles = StyleSheet.create({});
```
Comment (YilKanda) [inline ReservacionVuelos/src/components/TextTerms.js]: This part of the styles remember to put it in another file or if you don't use it, delete it.

Comment (Etneciv) [reply ReservacionVuelos/src/components/TextTerms.js]: Yes. thanks you, in next commit we'll correct this!

Comment (jbernalme) [reply ReservacionVuelos/src/components/TextTerms.js]: 8bcd3f67f0b33772e04824e6df56fd5877e5f8cc Done

**Review by Jonathanrrs — APPROVED** [review]
Excellent!

**Review by Kejoh — APPROVED** [review]
Good work with Text input

**Review by osvilla27 — APPROVED** [review]
Good job, dudes.

**Pull request merged**